mobile/src/main/java/org/openhab/habdroid/ui/OpenHABMainActivity.java

```diff
@@ -1289,19 +1288,15 @@ private void loadDrawerItems() {
             }
             mDrawerItemList.add(OpenHABDrawerItem.dividerItem());
         }
-//        mDrawerItemList.add(OpenHABDrawerItem.menuItem("Favorites", getResources().getDrawable(R.drawable.ic_star_grey600_36dp)));
```

Comment:
I think this line is the only "usage" of ic_star_grey600_36dp. Can you let this line here or also remove the star?

Reply:
As I don't like commented out code, as we can always go back in the history of git and look at the code, if we need to, I'll remove the star :)